src/ConfigUtil.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\oe_bootstrap_theme;
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Component\Utility\Crypt;
+
+/**
+ * Utility class with static methods related to configuration stuff.
```

Comment:
Don't describe what it has (so don't mention the static method), describe what its for.
Also, don't use "stuff", that helps with nothing to describe what this is for.